src/components/JourneyBlock.tsx

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+
+export interface JourneyEntry {
+    arrivalTime: string;
+    departureTime: string;
+    destinationStation: {'crs': string; 'displayName': string};
+    isFastestJourney: boolean;
+    journeyDurationInMinutes: number;
+    originStation: {'crs': string; 'displayName': string};
+    stationMessages: any[];
+    tickets: any[];
+}
+
+interface JourneyBlockProps {
+    journeyData: JourneyEntry;
+}
+
+const JourneyBlock = ({ journeyData }: JourneyBlockProps) => {
+    const { journeyDurationInMinutes: minutes, originStation, destinationStation } = journeyData;
+    const travelTime = `${Math.floor(minutes / 60)}hrs ${minutes % 60}mins`;
+
+    return (
+        <div className = "level is-size-5 box box-radius-large">
```

Comment:
I think we want a class name to add all this styling into instead of having 4 classes. But this will probably happen with your styling work next